src/Cardano/Shell/Types.hs

```diff
@@ -89,3 +87,468 @@ data CardanoFeature = CardanoFeature
     -- ^ What we call when we shut down the feature.
     }
 
+--------------------------------------------------------------------------------
+-- Cardano Mainnet Configuration
+--------------------------------------------------------------------------------
+
+mainnetConfiguration :: CardanoConfiguration
```

Comment:
Maybe move this into another module? It's not bad like this, but it might get tricky later on...